addons/hr_fleet/models/fleet_vehicle.py

```diff
@@ -9,7 +9,22 @@ class FleetVehicle(models.Model):
 
     mobility_card = fields.Char(compute='_compute_mobility_card', store=True)
 
-    @api.depends('driver_id', 'driver_id.user_ids.employee_id.mobility_card')
+    @api.depends('driver_id')
     def _compute_mobility_card(self):
         for vehicle in self:
-            vehicle.mobility_card = vehicle.driver_id.user_ids[:1].employee_id.mobility_card
+            employee = self.env['hr.employee']
+            if vehicle.driver_id:
+                employee = employee.search([('address_home_id', '=', vehicle.driver_id.id)])
+                if not employee:
+                    employee = employee.search([('user_id.partner_id', '=', vehicle.driver_id.id)])
+            vehicle.mobility_card = employee.mobility_card
```

Comment:
This could lead to a expected singleton error (even if it's not supposed to happen in a perfect world.